tvm/src/pass/adjust_buffer_binding.cc

```diff
@@ -0,0 +1,279 @@
+/*!
+ *  Copyright (c) 2019 by Contributors
+ * \file loop_partition.cc
```

Comment:
Incorrect name. Although we should use a script to automatically check this and modify this.